tests/console/clamav.pm

```diff
@@ -40,6 +40,12 @@ sub run {
     $self->select_serial_terminal;
 
     zypper_call('in clamav vim');
+
+    # Check Clamav version
+    # Jira ticket : SLE-16780 : upgrade Clamav SLE
+    zypper_call('info clamav');
+    record_info 'Check Clamav Version';
```

Comment:
Ben, you just recorded the pkg version here, correct?

I'd suggest you can check the version should >= 0.101

Reply:
yes, this is the version check, but I think it doesn't need to set as fail if the version is not yet upgraded. Becasue this is a Could Have case. I think mark the version record is fine currently. :-)

Reply:
Ok, but in this case, how can we tell openqa when the new package is there?

Reply:
yes, this is the point, I think it is a better way to check the version is newer. I will change it again.

Reply:
Agree, since we focus on the feature "upgrade to 0.103", we can merge the PR once the new pkg is there (or just marked it as soft fail?)

Reply:
Correct, I am adding the record_soft_fail if the newer package is not yet released. :)